platform-sdk/platform-apps/tests/StatsSigningTestingTool/src/main/java/com/swirlds/demo/stats/signing/TransactionPool.java

```diff
@@ -16,84 +16,33 @@
 
 package com.swirlds.demo.stats.signing;
 
-import static com.swirlds.common.utility.CommonUtils.hex;
-import static com.swirlds.logging.LogMarker.ADV_CRYPTO_SYSTEM;
-import static com.swirlds.logging.LogMarker.EXCEPTION;
-
-import com.swirlds.common.system.NodeId;
-import com.swirlds.common.system.transaction.Transaction;
-import com.swirlds.demo.stats.signing.algorithms.ExtendedSignature;
-import com.swirlds.demo.stats.signing.algorithms.SigningAlgorithm;
-import com.swirlds.demo.stats.signing.algorithms.X25519SigningAlgorithm;
-import edu.umd.cs.findbugs.annotations.NonNull;
-import edu.umd.cs.findbugs.annotations.Nullable;
-import java.security.SignatureException;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Objects;
 import java.util.Random;
-import org.apache.logging.log4j.LogManager;
-import org.apache.logging.log4j.Logger;
 
 /**
  * Provides pre-generated random transactions that are optionally pre-signed.
```

Comment:
Comment about "pre-signed" is no longer relevant

Reply:
removed